BenchmarkDotNet.Artifacts/results/Microsoft.ManagedZLib.Benchmarks.ManagedZLibBenchmark-report-github.md

```diff
@@ -0,0 +1,21 @@
+``` ini
```

Comment:
These results need to be excluded from this PR. We will post the results later after doing the comparison.